src/items/equipment/vali_water.json

```diff
@@ -0,0 +1,190 @@
+      "dex": {
+        "mod": ""
+      },
+      "hardness": {
+        "value": "2"
+      },
+      "hp": {
+        "max": "6",
+        "value": 6
+      },
+      "size": "medium",
+      "sturdy": false
+    },
+    "attuned": false,
+    "bulk": "",
+    "chatFlavor": "",
+    "consumableType": "serum",
+    "container": {
+      "contents": [],
+      "includeContentsInWealthCalculation": 1,
+      "isOpen": true,
+      "storage": [
+        {
+          "type": "slot",
+          "acceptsType": [
+            "spell"
+          ],
+          "affectsEncumbrance": false,
+          "amount": 0,
+          "subtype": "spellSlot",
+          "weightProperty": "bulk"
+        }
+      ]
+    },
+    "critical": {
+      "effect": "",
+      "parts": []
+    },
+    "damage": {
+      "parts": []
+    },
+    "damageNotes": "",
+    "description": {
+      "chat": "",
+      "gmnotes": "",
+      "short": "",
+      "unidentified": "",
+      "value": "<h3>Hybrid item</h3><p>The hot springs of Ouat Vali impart exceptional—potentially even permanent—vitality to bathers. Even water removed from its source and kept in carefully insulated vials provides some benefits, functioning in many ways like a magical serum. Drinking a vial of Vali water grants the imbiber a +1 insight bonus to attack rolls, skill checks, and saving throws; their abilities’ and spells’ saving throw DCs increase by 1; and they gain 1 temporary Resolve Point. After 1 hour, these effects end, the Resolve Point (if not already spent) disappears, and the user becomes @UUID[Compendium.sfrpg.conditions.JGFn4MrDZ6X5vrzU]{fatigued}. Drinking more than one vial of Vali water in a 24-hour period is dangerous, dealing 10d10 damage and giving the imbiber the@UUID[Compendium.sfrpg.conditions.NV4qXRDgb7zTNkW2]{ paralyzed} condition for 6d10 minutes unless they succeed at a DC 25 Fortitude save.</p><p>Genuine Vali water is exceedingly rare, as export is strictly prohibited by the Maratan government, and black markets are rife with counterfeits that range from merely useless to actively harmful.</p><p><br><em>Enable/Disable modifiers in the Modifiers tab.</em></p><h3>Serums</h3><p>Serums are vials of magic liquid that you can imbibe as a standard action or carefully trickle down the throat of a helpless or unconscious creature as a full action. Serums are normally 1 ounce of liquid (though it is possible to make serums with larger volumes, these are no more effective than standard serums), and lose potency if they are mixed with any other material. A serum can be used only once. Regardless of a serum’s actual item level, all serum vials have their hardness and Hit Points calculated as if they were 1st-level items.</p>"
+    },
+    "descriptors": [],
+    "duration": {
+      "units": "",
+      "value": null
+    },
+    "equippable": false,
+    "equipped": false,
+    "formula": "",
+    "identified": true,
+    "isActive": null,
+    "level": 16,
+    "modifiers": [
```

Comment:
Do modifiers work like this on consumables? Or do they need to be active?

Reply:
There is no activation button in sheet for consumables. So unless there's a better solution present, or one made. the consumable needs modifiers manually turned on/off